test/cpp/qps/client_callback.cc

```diff
@@ -83,8 +82,9 @@ class CallbackClient
   std::condition_variable shutdown_cv_;
   // Number of rpcs done after thread completion
   size_t rpcs_done_;
-  // Per Thread Queue of Context data pointers for running a RPC
-  std::vector<std::vector<std::unique_ptr<CallbackClientRpcContext>>> ctxs_;
+  // Vector of Context data pointers for running a RPC
+  std::vector<std::unique_ptr<CallbackClientRpcContext>> ctx_;
```

Comment:
People claim that I'm template-happy, but I'll point out that my comment helped bring this 4-way nested template down to 2-way. Cc: @ncteisen @a11r

Reply:
I can confirm this is true :)